### DISCUSSION TITLE
Color blue or black in page's changes management

### DISCUSSION THREAD

**Discussion by GlaDal** [opengisch/QField#4021] [Q&A]
Hi, 

Could you explain me the difference of colors in this page ? Why is there some attributes in blue and others in black ?
Thanks !

![image](https://user-images.githubusercontent.com/7015197/223995016-2b185bc0-e5b7-4edc-a104-6de329fda8ed.png)

**Comment by suricactus** [comment]
Hey, thanks for asking. Had to check myself too. :) The blues indicate the values are numbers OR booleans. The black is all the rest.

Red and green background are old and new values respectively.

**Reply by GlaDal** [reply]
Great thanks ! :-)
